pkg/controller/podgc/metrics.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package podgc
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	podGCController   = "pod_gc_collector"
+	deletingPods      = "deleting_pods_total"
```

Comment:
should we call it "force_deleted_pods_total" and "force_delete_pod_errors_total" ?

I would prefer to have word `force` somewhere in the name.

Reply:
I like those suggestions.

Reply:
That makes sense.